src/ontology/mondo-edit.obo

```diff
@@ -361087,32 +361054,52 @@ property_value: seeAlso "https://rarediseases.info.nih.gov/diseases/537/acute-no
 [Term]
 id: MONDO:0018875
 name: Li-Fraumeni syndrome
-def: "Li-Fraumeni syndrome (LFS) is a rare cancer predisposition syndrome characterized by the early-onset of multiple primary cancers such as breast cancer, soft tissue and bone sarcomas, brain tumors and adrenal cortical carcinoma (ACC)." [Orphanet:524]
+def: "A rare cancer predisposition syndrome characterized by the early-onset of multiple primary cancers such as breast cancer, soft tissue and bone sarcomas, brain tumors and adrenal cortical carcinoma (ACC)." [Orphanet:524]
+subset: clingen
 subset: ordo_disease {source="Orphanet:524"}
+synonym: "LFS" RELATED ABBREVIATION [OMIM:151623]
+synonym: "LFS1" RELATED ABBREVIATION [MONDO:Lexical, OMIM:151623]
+synonym: "LFS3" EXACT ABBREVIATION [MONDO:0012234, MONDO:Lexical, OMIM:609266]
 synonym: "Li Fraumeni syndrome" RELATED [GARD:0006902]
 synonym: "Li-Fraumeni familial cancer susceptibility syndrome" EXACT [NCIT:C3476]
 synonym: "Li-Fraumeni familiar cancer susceptibility syndrome" EXACT [DOID:3012, NCIT:C3476]
 synonym: "Li-Fraumeni syndrome" EXACT CLINGEN_PREFERRED [NCIT:C3476]
+synonym: "Li-Fraumeni syndrome" RELATED [OMIM:151623]
+synonym: "Li-Fraumeni syndrome caused by mutation in TP53" EXACT [MONDO:design_pattern]
+synonym: "Li-Fraumeni-like syndrome" RELATED [OMIM:151623]
+synonym: "sarcoma family syndrome of 51 and Fraumeni" RELATED [OMIM:151623]
 synonym: "sarcoma family syndrome of Li and Fraumeni" RELATED [GARD:0006902]
 synonym: "sarcoma, breast, leukaemia and adrenal gland syndrome" EXACT [DOID:3012]
 synonym: "SBLA syndrome" EXACT [DOID:3012]
+synonym: "Sbla syndrome" RELATED [OMIM:151623]
```

Comment:
the same synonym is both exact (from DO) and related (from OMIM)

Reply:
I will fix, thank you